app/src/lib/stores/pull-request-store.ts

```diff
@@ -139,9 +142,16 @@ export class PullRequestStore {
     api: API,
     repository: GitHubRepository
   ) {
-    const { name, owner } = getNameWithOwner(repository)
-    const open = await api.fetchAllOpenPullRequests(owner, name)
-    await this.storePullRequestsAndEmitUpdate(open, repository)
+    const bothPrLists = await Promise.all([
```

Comment:
You could destructure the returned array if you wanna be fancy, e.g.:

```suggestion
    const [repositoryPRs, parentPRs] = await Promise.all([
```